include/morphio/section.h

```diff
@@ -39,29 +39,23 @@ class Section: public SectionBase<Section>
     using PointAttribute = Property::Point;
 
   public:
-    /**
-       Depth first search iterator
-    **/
+    /// Depth firt search iterator
```

Comment:
firt -> first

It's not really a `search` iterator, so we can drop that for all of them

Reply:
yeah, fixed